Pets.API/Controllers/PetController.cs

```diff
@@ -3,18 +3,20 @@
 using Pets.BLL.Interfaces;
 using Pets.API.ViewModel;
 using Pets.BLL.Models;
+using Pets.DAL.Entities;
 
 namespace Pets.API.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class PetController : ControllerBase
     {
-        private readonly IPetService _service;
+        //it's temporary
+        private readonly IGenericService<Pet, PetEntity> _service;
```

Comment:
Remove temporary solution

Reply:
I'll remove this when i change controller